test/plugins/webhooks.test.js

```diff
@@ -970,6 +970,70 @@ describe('webhooks plugin test', () => {
                 });
             });
 
+            it('returns 201 on success for pipelines when mixed forward matching branch', () => {
+                const pMock1 = {
```

Comment:
Is this `pMock1` needed? It looks similar with `pipelineMock`.

Reply:
oh, deleted unnecessary mock.